src/azure-cli/HISTORY.rst

```diff
@@ -2,6 +2,9 @@
 
 Release History
 ===============
+**RDBMS**
```

Comment:
[nit] Please alphabetize this this with the rest of the services in this section.